nixos/modules/config/krb5/default.nix

```diff
@@ -92,278 +91,94 @@ in {
         '';
       };
 
-      libdefaults = mkOption {
-        type = with types; either attrs lines;
+      settings = mkOption {
         default = {};
-        apply = attrs: filterEmbeddedMetadata attrs;
-        example = literalExpression ''
-          {
-            default_realm = "ATHENA.MIT.EDU";
-          };
+        description = mdDoc ''
+          Structured contents of the `krb5.conf` file. See krb5.conf(5) for
+          details about configuration.
         '';
-        description = lib.mdDoc ''
-          Settings used by the Kerberos V5 library.
-        '';
-      };
-
-      realms = mkOption {
-        type = with types; either attrs lines;
-        default = {};
         example = literalExpression ''
```

Comment:
This example is probably a bit to big to be properly displayed on search.nixos.org

Reply:
I trimmed it down a bit, is that better?